applications/convection_diffusion_application/custom_elements/laplacian_element.cpp

```diff
@@ -13,7 +13,7 @@
 
 
 // External includes
-
+#include "utilities/geometry_utilities.h"
```

Comment:
do not include this header. it is only valid for tets and triangles, while the laplacian element is expected to work for any gometry